src/python/setup.sh.in

```diff
@@ -0,0 +1,3 @@
+# generated from ignition-math/setup.sh.in
+
+export PYTHONPATH=@CMAKE_INSTALL_PREFIX@/@IGN_PYTHON_INSTALL_PATH@:$PYTHONPATH
```

Comment:
This is being set to `install/lib/python/swig`, should it be `install/lib/python`?